### PR TITLE
Ggr change implement

### PR DESCRIPTION
**Pull request by Georges52**

### CODE REVIEW

**Comment by Georges52** [comment]
Updated version for the standard change implementation and valuemation handler

File: apiserver/.env.sample [hunk 1]
```diff
@@ -15,6 +15,11 @@ LOGGING_MAILHOST=
 JIRA_BASEURL=https://jira.example.org/atlassian/rest/api/
 JIRA_AUTH_USER=changeme
 JIRA_AUTH_PASS=changeme
+# Valuemation parameters
+VALUEMATION_BASEURL=https://int-oim.baloisenet.com/vmweb/services/api/execwf
```
Comment (matal0) [inline apiserver/.env.sample:19]: please replace with fake url

Comment (Georges52) [reply apiserver/.env.sample]: It is corrected now

**Review by vangual — CHANGES_REQUESTED** [review]
Test, unbedingt die url noch fixen fixen

**Review by vangual — APPROVED** [review]

**Pull request merged**